alicloud/common.go

```diff
@@ -454,7 +454,9 @@ func writeToFile(filePath string, data interface{}) error {
 		}
 	}
 
-	os.Remove(filePath)
+	if err := os.Remove(filePath); err != nil {
+		return err
+	}
```

Comment:
I'd actually prefer not doing `os.Remove` at all, since `ioutil.WriteFile` does most of the work already:

```
WriteFile writes data to a file named by filename. If the file does not exist, WriteFile creates it with permissions perm (before umask); otherwise WriteFile truncates it before writing.
```

However, I'm not sure if this will have any side effects since the function is widely used.